kennyedward/QuickCredit
# #165751907 Setup Mocha Test Framework to test all API Endpoints

#### What does this PR do?
It setups the mocha test framework for testing all API Endpoints in the application.

#### Description of Task to be completed?
Clone the repo,cd into the repo: ```cd quickcredit```
install dependency by running ```npm install```

#### How should this be manually tested?
- To test using the mocha framework: run ```npm test```
- To test using Postman: run ```npm run dev```, open Postman on your PC and navigate to ```http://localhost:7000/```. You should get a JSON Object: ```{ status: 200, message:"You're welcome" }```
#### Any background context you want to provide?
You should have node and npm installed on you

#### What are the relevant pivotal tracker stories?
#165751907
